datahub-web-react/src/app/entity/shared/constants.ts

```diff
@@ -57,6 +57,14 @@ export const EMPTY_MESSAGES = {
         title: 'Does not inherit from any terms',
         description: 'Terms can inherit from other terms to represent an "Is A" style relationship.',
     },
+    'contained by': {
+        title: 'Does not contained by any terms',
+        description: 'Terms can be contained by other terms to represent an "Has A" style relationship.',
```

Comment:
we can change the `...respresent an "Has A"...` to `...represent a "Has A"...`

also if you wouldn't mind making that "an" -> "a" change on line 54 as well that would be much appreciated :)